src/reconciliation.h

```diff
@@ -0,0 +1,335 @@
+#include <minisketch/include/minisketch.h>
+
+/** Static component of the salt used to compute short txids for transaction reconciliation. */
```

Comment:
Can you try to limit these lines to 80 or 100 chars to avoid wrapping.

Reply:
This particular line is less than 100 chars, so I assume you referred to the other ones which are longer than 100. I see a couple, gonna limit them.